pkg/app.go

```diff
@@ -84,12 +84,12 @@
 	fetchers := []fetchersPkg.Fetcher{
 		fetchersPkg.NewSlashingParamsFetcher(logger, appConfig, rpcs, tracer),
 		fetchersPkg.NewSoftOptOutThresholdFetcher(logger, appConfig, rpcs, tracer),
-		fetchersPkg.NewCommissionFetcher(logger, appConfig, rpcs, tracer),
+		fetchersPkg.NewCommissionFetcher(logger, appConfig.Chains, rpcs, tracer),
 		fetchersPkg.NewDelegationsFetcher(logger, appConfig, rpcs, tracer),
 		fetchersPkg.NewUnbondsFetcher(logger, appConfig, rpcs, tracer),
 		fetchersPkg.NewSigningInfoFetcher(logger, appConfig, rpcs, tracer),
 		fetchersPkg.NewRewardsFetcher(logger, appConfig, rpcs, tracer),
-		fetchersPkg.NewBalanceFetcher(logger, appConfig, rpcs, tracer),
+		fetchersPkg.NewBalanceFetcher(logger, appConfig.Chains, rpcs, tracer),
```

Comment:
Same as in another comment - we do not actually change anything here, we just pass only chains from config instead of all config to make it easier to mock.

Reply:
> could you confirm whether the initialization logic within NewCommissionFetcher and NewBalanceFetcher is strictly dependent only on the Chains data from appConfig

Yes, and it's the same for all other fetchers, except for the Price one.

Reply:
I will add another test for it, but essentially, it uses GetCoingeckoCurrencies that iterates over all config's chains.

Reply:
Done, can you check?